config.md

```diff
@@ -1,7 +1,7 @@
 # Container Configuration file
 
 The container's top-level directory MUST contain a configuration file called `config.json`.
-The canonical schema is defined in this document, but there is a JSON Schema in [`schema/config-schema.json`](schema/config-schema.json) and Go bindings in [`specs-go/config.go`](specs-go/config.go).
+The canonical schema is defined in this document, but there is a JSON Schema in [`schema/config-schema.json`](schema/config-schema.json) and Go bindings in [`specs-go/config.go`](specs-go/config.go). [**`Platform-specific`**](#platform) configuration schema are defined in platform-specific documents.
```

Comment:
There is already some documentation for this idea [here][1].  If you think it needs a mention in the top of this file, I'd consider just adding a link to the local `#platform-specific-configuration` (which actually links to the platform-specific documentation).

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc3/config.md#platform-specific-configuration